R/loadReg2.R

```diff
@@ -473,8 +475,9 @@ summarizeModel.loadReg2 <- function(load.model, ...) {
   
   # add the site.id as the leftmost column
   out$site.id <- getMetadata(load.model)@site.id
+  out$constituent <- getMetadata(load.model)@constituent
   site.id <- '.dplyr.var'
-  out <- select(out, site.id, everything())
+  out <- select(out, site.id, constituent, everything())
```

Comment:
warnings pop up when you build the package because of dplyr's non-standard eval (or something like that). It thinks that the col name "site.id" should be an object and then gives you a warning about how it's not a "global variable". So creating a dummy object named the same thing as your column prevents that